tests/manifold-map.cc

```diff
@@ -207,11 +253,14 @@ BOOST_AUTO_TEST_CASE_TEMPLATE (manifold_map_test_0, T, functionTypes_t)
 
   BOOST_CHECK (output->match_pattern());
 
- }
+}
 
 BOOST_AUTO_TEST_CASE_TEMPLATE (manifold_map_test_1, T, functionTypes_t)
 {
-  output = retrievePattern("manifold-map-1");
+  if (std::is_same<T, roboptim::EigenMatrixDense>::value)
+    output = retrievePattern("manifold-map-1-dense");
+  else
+    output = retrievePattern("manifold-map-1-sparse");
```

Comment:
Why not use the same output? If the problem is due to sparse matrix printing, you can just convert to dense matrices in both dense and sparse cases while printing.